comm/handlers.go

```diff
@@ -14,65 +18,296 @@ import (
 
 // Execute a command right away
 func Immediate(response http.ResponseWriter, request *http.Request) {
-	// TODO: Hide this ...
+
 	defer func() {
 		err := recover()
 		if err != nil {
-			response.Write([]byte("Server Error"))
+			HandleError(response, err)
 		}
 	}()
 
 	account, err := ValidateRequest(response, request)
 	if err != nil {
-		log.Fatal("Validate", err)
+		HandleError(response, err)
 	}
 
+	fmt.Println("Immediate Command")
```

Comment:
include some context when logging. A log that just says Immediate Command doesn't tend to be very usefull to tracing what is happening or going wrong.

Reply:
Scaffolding again. Should be deleted.